docs-src/go/how-to-customize-workflow-type-in-go.md

```diff
@@ -9,7 +9,7 @@ In Go, by default, the Workflow Type name is the same as the function name.
 
 To customize the Workflow Type, set the `Name` parameter with `RegisterOptions` when registering your Workflow with a [Worker](/go/how-to-develop-a-worker-in-go).
 
-<a class="dacx-source-link" href="https://github.com/temporalio/documentation-samples-go/blob/main/yourapp/worker/main_dacx.go">View source code</a>
+<a class="dacx-source-link" href="https://github.com/temporalio/documentation-samples-go/blob/add-go-schedule-sample/yourapp/worker/main_dacx.go">View source code</a>
```

Comment:
Is this intentional?

Reply:
In the future, I will hold off on yarn gen until after it gets through a review. But yeah, there will be links to source code before snippets